### ISSUE TITLE
Selected text Inconstancies in the 0.9.0 Clarity Sketch template 

### ISSUE THREAD

**Issue by tonycgil** [vmware/clarity#1073]
[ x] bug
[ ] feature request
[ ] enhancement
```
Inconstancies found in the 0.9.0 Clarity Sketch template:

Dropdown - text on selection #4A4A4A Metropolis Regular
Wizard - text on selection #000000 Metropolis Regular
Tree - text on selection #565656 Metropolis Regular
Stack view - text on selection #565656 Metropolis Medium

Expected behavior:
They should be the same


* **Clarity version:** 
0.9.0

**Comment by reddolan** [comment]
@tonycgil I updated the selection font color in Wizard to #313131 not #000000. 

![image](https://user-images.githubusercontent.com/1858226/27239882-6113ecb6-5298-11e7-8b33-1c4a347eec74.png)

**Comment by reddolan** [comment]
Changes made and merged into #1077

**Comment by tonycgil** [comment]
@reddolan thanks!
